ToolBox/TradingEconomicsDataDownloader/TradingEconomicsCalendarDownloader.cs

```diff
@@ -190,5 +192,247 @@ public override Task<string> Get(DateTime startUtc, DateTime endUtc)
+            public string DateSpan { get; set; }
+
+            /// <summary>
+            /// Importance of a TradingEconomics information
+            /// </summary>
+            [JsonProperty(PropertyName = "Importance")]
+            public TradingEconomicsImportance Importance { get; set; }
+
+            /// <summary>
+            /// Time when new data was inserted or changed
+            /// </summary>
+            [JsonProperty(PropertyName = "LastUpdate"), JsonConverter(typeof(TradingEconomicsDateTimeConverter))]
+            public DateTime LastUpdate { get; set; }
+
+            /// <summary>
+            /// Value reported in the previous period after revision
+            /// </summary>
+            /// <remarks>
+            /// If there is no revision field remains empty
+            /// </remarks>
+            [JsonProperty(PropertyName = "Revised")]
+            public string Revised { get; set; }
+
+            /// <summary>
+            /// Country's original name
+            /// </summary>
+            [JsonProperty(PropertyName = "OCountry")]
+            public string OCountry { get; set; }
+
+            /// <summary>
+            /// Category's original name
+            /// </summary>
+            [JsonProperty(PropertyName = "OCategory")]
+            public string OCategory { get; set; }
+
+            /// <summary>
+            /// Unique ticker used by Trading Economics
+            /// </summary>
+            [JsonProperty(PropertyName = "Ticker")]
+            public string Ticker { get; set; }
+
+            /// <summary>
+            /// Unique symbol used by Trading Economics
+            /// </summary>
+            [JsonProperty(PropertyName = "Symbol")]
+            public string TESymbol { get; set; }
+
+
+            public TradingEconomicsCalendar Parse()
+            {
+                return new TradingEconomicsCalendar
+                {
+                    CalendarId = CalendarId,
+                    EndTime = EndTime,
+                    Country = Country,
+                    Category = Category,
+                    Event = Event,
+                    Reference = Reference,
+                    Source = Source,
+                    Actual = ParseDecimal(Actual),
+                    Previous = ParseDecimal(Previous),
+                    Forecast = ParseDecimal(Forecast),
+                    TradingEconomicsForecast = ParseDecimal(TradingEconomicsForecast),
+                    DateSpan = DateSpan,
+                    Importance = Importance,
+                    LastUpdate = LastUpdate,
+                    Revised = ParseDecimal(Revised),
+                    OCountry = OCountry,
+                    OCategory = OCategory,
+                    Ticker = Ticker,
+                    TESymbol = TESymbol,
+
+                    IsPercentage = Actual.EndsWith("%") || Previous.EndsWith("%") || Forecast.EndsWith("%") || TradingEconomicsForecast.EndsWith("%")
```

Comment:
The `||` operations to defined `IsPercentage` worries me, how will the user be able to determine which value is a percentage or not? 
It seems these values are all referring to the same 'thing', have we seen a case where some are a `%` and some are not? 
Could we normilize them before hand so that all values are a percentage or not?